docs/overview/diy.md

```diff
@@ -19,7 +19,7 @@ Although we use custom hardware to improve our commercial product and enable add
 
 * Case option A (internal storage)
   * Custom case (e.g. made of acrylic like [Digital Garage's hack0](https://github.com/dgarage/hack0-hardware))
-  * PCIe SSD adapter, either our own [minimal adapter](https://shop.shiftcrypto.ch/en/products/compact-m2-to-pcie-adapter-15/) or Pine64's [standard adapter](https://store.pine64.org/?product=rockpro64-pci-e-x4-to-m-2ngff-nvme-ssd-interface-card)
+  * PCIe SSD adapter, either our a minimal adapter or Pine64's [standard adapter](https://store.pine64.org/?product=rockpro64-pci-e-x4-to-m-2ngff-nvme-ssd-interface-card)
```

Comment:
remove `a`: "... either our minimal adapter or the Pine64's ..."

Reply:
Done